.github/workflows/CI.yml

```diff
@@ -17,14 +17,9 @@ jobs:
         with:
           node-version: 20
 
-      - name: Install Dependencies
+      # As our package has a postInstall script, we dont need a explicit job for that
+      - name: Install & Build Dependencies
         run: npm install 
-      
-      - name: Build 3Dmol.js
-        run: |
-          npm run build:dev
-          npm run build:prod
-          npm run generate:tests
```

Comment:
Removed the Build job, which ~saves us 30 seconds in CI time

But why this change? Our `package.json` has a `"postInstall"` script here, https://github.com/3dmol/3Dmol.js/blob/master/package.json#L33 which runs `npm run build` upon installation of dependencies. Till now we were running a redundant job, hence removed the same